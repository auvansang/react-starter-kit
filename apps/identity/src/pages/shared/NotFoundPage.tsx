import { Box, Link } from '@mui/material';

import logo from 'assets/logo.png';

const NotFoundPage = () => {
  return (
    <Box
      sx={{
        height: '100vh',
        display: 'flex',
        alignItems: 'center',
        justifyContent: 'center',
      }}
    >
      <Box sx={{ flex: 1 }}>
        <Link href="/">
          <Box
            sx={{
              mb: 8,
              height: 84,
              backgroundImage: `url(${logo})`,
              backgroundRepeat: 'no-repeat',
              backgroundPosition: 'center center',
              backgroundSize: 'contain',
            }}
          ></Box>
        </Link>
      </Box>
    </Box>
  );
};

export default NotFoundPage;
