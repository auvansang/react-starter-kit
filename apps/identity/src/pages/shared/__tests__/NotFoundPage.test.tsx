import renderer from 'react-test-renderer';

import NotFoundPage from '../NotFoundPage';

it('It rendered successfully', () => {
  const component = renderer.create(<NotFoundPage />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
