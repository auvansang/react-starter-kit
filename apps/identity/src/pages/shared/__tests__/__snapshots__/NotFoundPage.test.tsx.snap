// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It rendered successfully 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-1uygidu{height:100vh;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;}",
      }
    }
    data-emotion="css 1uygidu"
  />,
  <div
    className="MuiBox-root css-1uygidu"
  >
    <style
      dangerouslySetInnerHTML={
        Object {
          "__html": ".css-1rr4qq7{-webkit-flex:1;-ms-flex:1;flex:1;}",
        }
      }
      data-emotion="css 1rr4qq7"
    />
    <div
      className="MuiBox-root css-1rr4qq7"
    >
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-12u91cq-MuiLink-root{-webkit-text-decoration:underline;text-decoration:underline;text-decoration-color:rgba(25, 118, 210, 0.4);}.css-12u91cq-MuiLink-root:hover{text-decoration-color:inherit;}",
          }
        }
        data-emotion="css 12u91cq-MuiLink-root"
      />
      <style
        dangerouslySetInnerHTML={
          Object {
            "__html": ".css-1ps4owl-MuiTypography-root-MuiLink-root{margin:0;color:#1976d2;-webkit-text-decoration:underline;text-decoration:underline;text-decoration-color:rgba(25, 118, 210, 0.4);}.css-1ps4owl-MuiTypography-root-MuiLink-root:hover{text-decoration-color:inherit;}",
          }
        }
        data-emotion="css 1ps4owl-MuiTypography-root-MuiLink-root"
      />
      <a
        className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
        href="/"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        <style
          dangerouslySetInnerHTML={
            Object {
              "__html": ".css-1n6uv42{margin-bottom:64px;height:84px;background-image:url(logo.png);background-repeat:no-repeat;-webkit-background-position:center center;background-position:center center;-webkit-background-size:contain;background-size:contain;}",
            }
          }
          data-emotion="css 1n6uv42"
        />
        <div
          className="MuiBox-root css-1n6uv42"
        />
      </a>
    </div>
  </div>,
]
`;
